tests/test_playbooks/tasks/host.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: "Ensure host '{{ host_name }}' is {{ host_state }}"
   vars:
-    host_name: "test-host"
+    host_name: "test-host.{{ host.domain }}"
```

Comment:
this makes `tasks/host.yml` dependant on `vars/host.yml` and makes it hard for the user to override the domain of one host. how about we omit `host_name` here completely and explicitly set it in every call in `test_playbooks/host.yml`?

Reply:
true, moving it there

Reply:
will also remove host_state hardcoded to present. State is present by default anyway